### ISSUE TITLE
Add meta tag info to support richer URL unfurling

### ISSUE THREAD

**Issue by kdoh** [metabase/metabase#3374]
Right now when you post Metabase links in Slack or other places that use meta and open graph tags to "unfurl" the URL we don't really provide any rich metadata for them to use which results in pretty lame and hard to parse URLs. Ideally we'd be able to provide the question or dashboard title, author and description, and in the case of questions, the question viz type, as metadata so that URLs are richer and more dynamic.

**Comment by puppycodes** [comment]
please yes for this +1

**Comment by designedbyclu** [comment]
Any update about this?

**Comment by james-freec** [comment]
We can not embed it to notion or any platform required open grapth

**Comment by albertoperdomo** [comment]
Was this written with public or private URLs in mind?

**Comment by renchap** [comment]
It would be great for public links, but some other software have it working for private links as well, as long as you configured the Slack integration. Slack pings the app when a link with a matching domain is detected, and Metabase could reply with preview data, even if the links are private, as the Slack app has credentials.

**Comment by albertoperdomo** [comment]
@renchap Yes, I think that would be helpful indeed. Since that requires a different solution via deep integrations, I think we would track those feature requests separately though. Probably individually too, so that we can track demand (Slack vs Notion vs something else). Makes sense?

**Comment by systemmonkey42** [comment]
Hi,  Its been years.  Is this even on the road map?

Thanks
